docs/operators/rest.md

```diff
@@ -0,0 +1,76 @@
+---
+title: REST on GraphQL
+description: Exposing REST endpoints on top of GraphQL via @rest directive.
+---
+
+Tailcall's `@rest` directive exposes standard REST endpoints, enabling the fetching of data from REST APIs directly through GraphQL queries.
+
+This guide will show you how to use the `@rest` directive to create a RESTful service over a GraphQL layer for retrieving product information.
```

Comment:
An intro about it's need:

Every one needs orchestration for APIs but everyone can't move to GraphQL and there are valid reasons for not moving to GraphQL. This feature allows people to building the Tailcall DSL, leveraging the GraphQL capabilities at compile time and generating REST endpoint for actual use instead. REST is a well-known standard so works with traditional API Infrastructure - CDNs Gateways etc.